src/main/java/server/router/Route.java

```diff
@@ -0,0 +1,48 @@
+package server.router;
+
+public enum Route {
+    HOME("/"),
+    FORM("/form"),
+    METHOD_OPTIONS("/method_options"),
+    REDIRECT("/redirect"),
+    IMAGE_JPEG("/image.jpeg"),
+    IMAGE_PNG("/image.png"),
+    IMAGE_GIF("/image.gif"),
+    FILE("/file1"),
+    TEXT_FILE("/text-file.txt"),
+    PARAMETERS("/parameters"),
+    LOGS("/logs"),
+    LOG("/log"),
+    THESE("/these"),
+    REQUESTS("/requests"),
+    PARTIAL_CONTENT("/partial_content.txt"),
+    PATCH_CONTENT("/patch-content.txt");
```

Comment:
why did you feel it was necessary to wrap the files that are served in as a list of enum? Could you not have just served these through the public directory?